workflow/scripts/filter_primers.rs

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env rust-script
```

Comment:
```suggestion
```
you don't actually need the shebang when using rust-scripts in snakemake contexts.